CleanWeather/Extensions/ErrorExtension.swift

```diff
@@ -12,13 +12,18 @@ struct AppError: Error {
     let message: String
 }
 
+struct UnitTestError: Error {}
+
 extension Error {
     var userFriendlyMessage: String {
         if self is DecodingError {
             return R.string.localizable.decodingError()
         } else if let appError = self as? AppError {
             return appError.message
+        } else if self is UnitTestError {
+            return R.string.localizable.unitTestError()
```

Comment:
do not localize strings that aren't going to an app. Unit Test Error should be only in Test target